src/webhooks/registry.ts

```diff
@@ -354,11 +356,71 @@ export interface WebhookProcessParams extends AdapterArgs {
   rawBody: string;
 }
 
+const statusTextLookup: {[key: string]: string} = {
```

Comment:
While I'm a fan of this, I wonder if it'd be worth making it more generic, since it seems pretty useful all around. Not sure if we'd use it anywhere else though!

Reply:
The only other place in non-test code that we could use something like it is https://github.com/Shopify/shopify-api-node/blob/3e251da3b9d60810a9b33cef7bedafd58f10f8cd/src/auth/oauth/oauth.ts#L80-L81